src/components/__tests__/Tabs-test.js

```diff
@@ -185,6 +185,27 @@ describe('react-tabs', () => {
       expect(wrapper.childAt(2).text()).toBe('Hello Bar');
       expect(wrapper.childAt(3).text()).toBe('Hello Baz');
     });
+
+    it('should not clone non tabs element', () => {
```

Comment:
Nice to figure out how to test this 👍
You can just add `/* eslint-disable react/no-multi-comp */` at the top of the test to disable the lint.